### DISCUSSION TITLE
Want to run phi ai assistant but it wont create workspace and recognize commands

### DISCUSSION THREAD

**Discussion by Traveltainer007** [agno-agi/agno#95] [General]
Ive installed and checked all the folders and files and gpt even said that all the files look fine and in order 

I have gitbash and python312 and win10
Everything is updated too 

This is the second time im trying to do this in 4 days 

Before i was able to add 38 functions but then they all disappeared and theres no workspace

Ive run phi ws create and it doesnt recognize it etc

**Comment by ashpreetbedi** [comment]
@Traveltainer007 thanks for reaching out and sorry you're facing this error. I'm here to help. If you want real-time help im on the discord to. Now lets fix this:

1. Is the virtual env active?
2. If you created a workspace using `phi ws create`, then please can you `phi ws setup` in that directory and try running `phi ws up`
3. Can you share the output of `phi config -a`
4. Can you share which command isnt working

And lets take it from there :) 

Im here to help

**Comment by Traveltainer007** [comment]
It appears that the phi command is not recognized, which could indicate that the Phi AI Assistant CLI is not installed or not added to the system's PATH

 Volume in drive C is icyou
 Volume Serial Number is 027D-D38D

 Directory of C:\Users\user\TraveltainerworldAI\phidata\phi

02/10/2024  09:46 AM    <DIR>          .
02/10/2024  09:46 AM    <DIR>          ..
02/10/2024  09:46 AM    <DIR>          ai
02/10/2024  09:46 AM    <DIR>          api
02/10/2024  09:46 AM    <DIR>          app
02/10/2024  09:46 AM    <DIR>          assistant
02/10/2024  09:46 AM    <DIR>          aws
02/09/2024  09:06 AM             4,827 base.py
02/10/2024  09:46 AM    <DIR>          cli
02/09/2024  09:06 AM             1,244 constants.py
02/10/2024  09:46 AM    <DIR>          docker
02/10/2024  09:46 AM    <DIR>          document
02/10/2024  09:46 AM    <DIR>          embedder
02/10/2024  09:46 AM    <DIR>          file
02/10/2024  09:46 AM    <DIR>          infra
02/10/2024  09:46 AM    <DIR>          k8s
02/10/2024  09:46 AM    <DIR>          knowledge
02/10/2024  09:46 AM    <DIR>          llm
02/10/2024  09:46 AM    <DIR>          memory
02/09/2024  09:06 AM                 0 py.typed
02/10/2024  09:46 AM    <DIR>          resource
02/10/2024  09:46 AM    <DIR>          storage
02/10/2024  09:46 AM    <DIR>          table
02/10/2024  09:46 AM    <DIR>          task
02/10/2024  09:46 AM    <DIR>          tools
02/10/2024  09:46 AM    <DIR>          utils
02/10/2024  09:46 AM    <DIR>          vectordb
02/10/2024  09:46 AM    <DIR>          workspace
02/09/2024  09:06 AM                 0 __init__.py
               4 File(s)          6,071 bytes
              25 Dir(s)  11,019,317,248 bytes free

(myenv) C:\Users\user\TraveltainerworldAI\phidata\phi> Directory of C:\Users\user\TraveltainerworldAI\phidata\phi
'Directory' is not recognized as an internal or external command,
operable program or batch file.

(myenv) C:\Users\user\TraveltainerworldAI\phidata\phi>02/10/2024  09:46 AM    <DIR>          .
The system cannot find the file specified.

(myenv) C:\Users\user\TraveltainerworldAI\phidata\phi>02/10/2024  09:46 AM    <DIR>          assistant
The system cannot find the file specified.

C:\Users\user\TraveltainerworldAI\phidata\phi>phi was create
C:\Users\user\TraveltainerworldAI\phidata\phi was unexpected at this time.
(myenv) C:\Users\user\TraveltainerworldAI\phidata\phi>cd..

(myenv) C:\Users\user\TraveltainerworldAI\phidata>phi ws create
'phi' is not recognized as an internal or external command,
operable program or batch file.


(myenv) C:\Users\user\TraveltainerworldAI\phidata>

**Comment by ashpreetbedi** [comment]
@Traveltainer007 

in the phidata directory, please create a virtualenv:

```bash
python3 -m venv aienv
aienv/scripts/activate
```

install phidata + other libraries you need:

```bash
pip install -U phidata openai
```

Then `phi` command will be available. I'm here to help.
